cli/icon-generator.js

```diff
@@ -70,21 +75,26 @@ function createSvg(entry) {
 
 }
 
-function createSvgs(svgs) {
-	svgs.forEach((entry) => {
-		createSvg(entry);
+function createSvgs(categories) {
+	categories.forEach((category) => {
+		category.svgs.forEach((name) => {
+			createSvg(category.name, name);
+		});
 	});
 }
 
-function createLoader(svgs) {
+function createLoader(categories) {
 
 	let template = '// auto-generated\n' +
 		'export async function loadSvg(icon) {\n' +
 		'\tswitch (icon) {\n';
-	svgs.forEach((svg) => {
-		template += `\t\tcase 'd2l-${svg.category}:${svg.name}':
-			return await import('./${svg.category}/${svg.name}.js');\n`;
+	categories.forEach((category) => {
+		category.svgs.forEach((name) => {
+			template += `\t\tcase 'd2l-${category.name}:${name}':
+				return await import('./${category.name}/${name}.js');\n`;
```

Comment:
I think `await` and `async` (on `loadSvg`) are not needed here.

Reply:
You're correct -- fixed!